README.md

```diff
@@ -49,6 +56,21 @@ make
 make test
 ```
 
+Optionally, you can install [DeepMind Lab](https://deepmind.com/blog/open-sourcing-deepmind-lab) and [Project Malmo](https://www.microsoft.com/en-us/research/project/project-malmo).
```

Comment:
Suggest to change the links to github links which is more direct for users who want to install the packages.
https://github.com/deepmind/lab
https://github.com/Microsoft/malmo